File: Kattis/Java/Aaah.java
/**
*Solution for kattis problem source: https://open.kattis.com/problems/aaah
*/

import java.util.Scanner;

class Aaah {
  public void handleInput(){
    Scanner kattio = new Scanner(System.in);
    String first = kattio.nextLine();
    String second = kattio.nextLine();
    kattio.close();
    handleOutput(first, second);
  }
  public void handleOutput(String first, String second) {
    if (first.length() >= second.length()) {
      System.out.println("go");
    } else {
      System.out.println("no");
    }
  }
  public static void main(String[] args) {
    Aaah thisInstance = new Aaah();
    thisInstance.handleInput();
  }
}

File: Go/Aaah.go
package main

import (
	"fmt"
)

// Solution for kattis: https://open.kattis.com/problems/aaah

func in() {
	var a, b string
	fmt.Scanf("%s", &a)
	fmt.Scanf("%s", &b)
	out(a, b)
}

func out(a, b string) {
	if len(a) >= len(b) {
		fmt.Println("go")
	} else {
		fmt.Println("no")
	}
}

func main() {
	in()
}
